libgnucash/engine/Account.cpp

```diff
@@ -3242,15 +3242,18 @@ xaccAccountGetName (const Account *acc)
     return GET_PRIVATE(acc)->accountName;
 }
 
+std::vector<const Account*>
+gnc_account_get_path (const Account *account)
```

Comment:
`path` might be a little confusing in this context, how about `ancestry` or `all_parents`?